docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -160,9 +160,14 @@ whether an alert is generated.
 |threshold |Object a|Defines the field and threshold value for when alerts
 are generated, where:
 
-* `field` (string, required): The field on which the threshold is applied. If
+* `cardinality_field` (string, required): The field on which the cardinality is applied. If
```

Comment:
Thanks @spong! This is the current behavior, but the field should not be required, and `cardinality_value` will only be required if `cardinality_field` is supplied. Will be updating this in an upcoming PR. I can change this here when that goes in...

Reply:
@madirey are we good to update this now with the latest fixes that made it in?

Reply:
@spong Yeah, the schema looks a bit different now. Happy to update on Monday, or if you'd rather, I pasted the new schema below.

![image](https://user-images.githubusercontent.com/611653/110258831-8c31b800-7f72-11eb-91d1-3d2e4d01a198.png)

Reply:
Yeah if you could update that'd be great. I think this is the last change needed then we can merge 👍